src/components/Canvas.tsx

```diff
@@ -1,28 +1,42 @@
 import React from 'react';
 import '../styles/Canvas.scss';
-import {IInfographicsSection} from '../types/TInterfaceSection';
-import InfographicsSection from './InfographicsSection';
+import {IInfographicsSection} from '../types/IInfographicsSection';
+import {IInfographicsDetails} from '../types/IInfographicsDetails'
+import SectionRow from './SectionRow'
 
 interface IProps {
-    infographicsSections: IInfographicsSection[],
-    selectedSectionId: number | null,
-    handleDeleteSection: (id: number) => void,
+    infographicsDetails: IInfographicsDetails,
+    selectedSection: IInfographicsSection | null,
+    handleToggleSelectSection: (section: IInfographicsSection) => void,
+    handleDeleteSection: (section: IInfographicsSection) => void,
 }
 
 const Canvas: React.FC<IProps> = (props) => {
 
-    const {infographicsSections, selectedSectionId, handleDeleteSection} = props;
+    const {infographicsDetails, selectedSection, handleToggleSelectSection, handleDeleteSection} = props;
```

Comment:
nekde mas `;` nekde ne, mela by jsi to drzet jednotne, ale asi to timhle nemusis blokovat, to je REPLACE ALL hned, ale udrzoval bych jednotny code-style